src/collections/_documentation/sdks/javascript/index.md

```diff
@@ -17,4 +17,14 @@ $ npm install @sentry/browser
 
 {%- include common/getting-started.md 
 sdk_name=page.title
-install_content=__install_content -%}
\ No newline at end of file
+install_content=__install_content -%}
+
+{%- capture __config_content -%}
+
+```Sentry.init({ dsn: '___PUBLIC_DSN___' });```
+
+{%- endcapture -%}
+
+{%- include common/getting-started.md 
+sdk_name=page.title
+install_content=__config_content -%}
```

Comment:
You should only have the `include` once on the bottom.
Also, this should be `config_content=__config_content` not `install_content=__config_content`

Reply:
fixed config_content=__config_content, thank you.

I'm not sure I understand the include comment? Perhaps I'm not coding this correctly, but the first include is for install, the second is for configuration content, so doesn't it need to be listed twice? If I understand the structure (which I may not):

{%- capture <thing_we're_calling_it> -%}
<content_to_include>
{%- endcapture -%}

{%- include <dir/filename>
sdk_name=page.title
<thing_we're_calling_it>=<thing_we're_calling_it> %}

And we need this for each of the items we're including?

Reply:
I fixed it in this commit 
https://github.com/getsentry/sentry-docs/pull/1746/commits/2d27304e77c98550ca6396bd51f9bca0167a8d00

We only need to have the include once with multiple variables.

Reply:
Also for the DSN selectors and rest to work properly we need a spacing here
https://github.com/getsentry/sentry-docs/pull/1746/commits/c1c230483d329dd4c3594ec2f9acbf36ef120436

Reply:
Thank you. Take a look a the latest - it renders properly on my local server.